analyzers/tests/SonarAnalyzer.UnitTest/TestCases/MethodsShouldNotHaveTooManyLines.LocalFunctions.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+
+namespace Tests.Diagnostics
+{
+    public class Foo
+    {
+        public void Bar(int i) // Compliant, because we do not count the local static functions lines.
+        {
+            Console.WriteLine(i);
+
+            void LocalFunction(int x)
+            {
+                Console.WriteLine(x);
+            }
+
+            static void StaticLocalFunction()
```

Comment:
What about the static local function itself? Shouldn't we check then the line count at the function level?

Do we plan to do this as a baby step and add an issue to implement the check later?

Reply:
Yes! The idea is that we'll do this (count the static local functions separately) for all the rules affected in this [issue](https://github.com/SonarSource/sonar-dotnet/issues/5625) in another issue (I'll open it today, once I put the last rule in review).